fleetspeak/src/client/clienttestutils/clienttestutils.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clienttestutils
+
+import (
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+
+	fspb "github.com/google/fleetspeak/fleetspeak/src/common/proto/fleetspeak"
+)
+
+// WriteServiceConfig saves the given config to a test location.
+func WriteServiceConfig(t *testing.T, dirpath, filename string, cfg *fspb.SignedClientServiceConfig) {
```

Comment:
We should avoid passing around the testing.T object. Having this function return an error is preferred.

https://golang.org/doc/faq#testing_framework

Reply:
Done.